launchable/utils/no_build.py

```diff
@@ -0,0 +1,2 @@
+NO_BUILD_BUILD_NAME = "anonymous"
```

Comment:
[nits]
I think `anonymous` is used for person and not for thing or idea.
How about `no_named_build` or `nameless` or `unnamed`?